utils/split.pl

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+# NOTE: needs libpdf-api2-perl
+use PDF::API2;
+
+sub usage
+{
+  my ($stream, $status) = @_;
+
+  print $stream "Usage: $0 <index_file> <pdf_file> <output_directory>\n";
+
+  exit $status;
+}
+
+usage(*STDERR, 1) unless $ARGV[2];
+
+my ($index,$pdfInput,$outputDir) = @ARGV;
+
+print "index: $index\n";
+print "pdfInput: $pdfInput\n";
+print "outputDir: $outputDir\n";
+
+open(IDX, "<$index") || die "Could not open index file $index: $!\n";
+
+#die "PDF file $pdfInput is not a readable file\n" unless -f $pdfInput && -r $pdfInput;
+
+die "Output directory $outputDir does not exist\n" unless -d $outputDir;
+
+my $sourcepdf = PDF::API2->open($pdfInput) || die "Could not open PDF file $pdfInput: $!\n";
+
+while (<IDX>)
+{
+  chop;
+  next if ( ! m/^(.*),(.*),([^,]*)$/ );
```

Comment:
Does this handle quoted titles like those in https://raw.githubusercontent.com/aspiers/book-indices/master/RealBk2.csv?

Reply:
Commas in titles are indeed ok:
```
$ perl -ne 'print "T:[$1]  P:$2-$3" if ( m/^(.*),(.*),([^,]*)$/ )'
song,0,
T:[song]  P:0-
song,with,comma,4,5
T:[song,with,comma]  P:4-5
"quoted, song with comma",7,10
T:["quoted, song with comma"]  P:7-10
```

I'll look into handling the missing end page, thanks for the review

Reply:
Pushed handling of missing end, comments and empty lines